docs/sanic/middleware.rst

```diff
@@ -14,8 +14,8 @@ There are two types of middleware: request and response. Both are declared
 using the `@app.middleware` decorator, with the decorator's parameter being a
 string representing its type: `'request'` or `'response'`.
 
-* Request middleware receives only the `request` as argument.
-* Response middleware receives both the `request` and `response`.
+* Request middleware receives only the `request` as an argument and are executed in the order they were added.
```

Comment:
Change  "and are" to "and is"

Reply:
I think 'middleware' is used as a plural in this context. See "the order _they_ were added", implying plural. So "and are" is more correct in this sentence.

For this reason I've personally started using the term _middlewares_ to mean plural, and _middleware_ to mean singular, to reduce the ambiguity.